### PR TITLE
remove cancel button and add backtolist button

### PR DESCRIPTION
**Pull request by SOLenG**

### CODE REVIEW

File: MediaAdminBundle/Resources/public/coffee/view/MediaUploadView.coffee [hunk 1]
```diff
@@ -79,7 +81,7 @@ MediaUploadView = OrchestraView.extend(
     @r.on 'fileSuccess', (file, message) ->
       $self = $('.flow-file-' + file.uniqueIdentifier)
       $self.find('.flow-file-progress').text '(' + $('#uploadCompleted').text() + ')'
-      $self.find('.flow-file-pause, .flow-file-resume').remove()
+      $self.find('.flow-file-pause, .flow-file-resume, .flow-file-cancel').remove()
```
Comment (ngilain) [inline MediaAdminBundle/Resources/public/coffee/view/MediaUploadView.coffee:84]: Does this preserve the cancel button during upload ?

**Comment by nicolasThal** [comment]
:+1:

**Comment by Thiblef** [comment]
:+1:

File: MediaAdminBundle/Resources/public/coffee/view/MediaUploadView.coffee (hunk 1 above)
Comment (SOLenG) [inline MediaAdminBundle/Resources/public/coffee/view/MediaUploadView.coffee:84]: @itkg-ngilain Yes, the cancel button is remove only on the end upload.

**Pull request merged**